src/pipeline.ts

```diff
@@ -816,6 +938,68 @@ function snakeCaseKeys<T = unknown>(obj: T, sep = '-'): T {
   return result as any;
 }
 
+/**
+ * Multiple stages that are deployed in parallel
+ *
+ * A `Wave`, but with addition GitHub options
+ *
+ * Create with `GitHubWorkflow.addWave()` or `GitHubWorkflow.addGitHubWave()`, do not construct directly
+ *
+ * @hideconstructor
```

Comment:
why `@hideconstructor`? what does it do and why do we need it here. can't we make the constructor private if we don't want it public?

Reply:
It was a futile attempt at getting the JSII doc builder to hide the constructor. I also couldn't find a clean way to make it private. I'll remove this, it shouldn't have made it into the push.